### DISCUSSION TITLE
Demo pyodide with Websockets

### DISCUSSION THREAD

**Discussion by pak-man** [pyodide/pyodide#3073] [Show and tell]
Hello,

i´ve whipped a small demo how to use pyodide with JS Websockets, see here:
[Demo-Websockets-Pyodide](https://starboard.gg/ACBIAS/Demo-Websockets-Pyodide-nbamPAL)

**Comment by chrysn** [comment]
The demo works with starboard, but doing the same with current pyodide requires using a changed API.

An example of how pyodide 0.19 code is adjusted to WebSocket operations that work on 0.22.1 is in https://github.com/chrysn/aiocoap/issues/258#issuecomment-1478514538 and its following comment.
